plugwise_usb/nodes/circle.py

```
@@ -647,22 +659,50 @@ async def set_relay(self, state: bool) -> bool:
+            result = True
+        
         _LOGGER.debug(
             "Failed to restore relay state from cache for node %s, try to request node info...",
             self._mac_in_str,
         )
         if await self.node_info_update() is None:
-            return False
-        return True
+            result = False
+
+        if (cached_relay_lock = self._get_cache(CACHE_RELAY_LOCK)) is not None:
```

Ruff (0.11.9)

SyntaxError: Expected ')', found '='

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Invalid annotated assignment target

---

SyntaxError: Expected an expression

---

SyntaxError: Unexpected indentation

---

Blank line contains whitespace

(W293)